pacta/build_with_tag.sh

```diff
@@ -54,7 +54,12 @@ do
     fi
 done
 
-existing_images="$(docker images -q '2dii_pacta')"
+if (! docker images > /dev/null 2>&1 )
+then
+  red "The docker daemon does not appear to be running." && exit 1
+fi
```

Comment:
Thanks @cjyetman! 
I pasted your solution.

The only issue I see is in the scenario when the user has no image but the daemon is indeed working. That situation is possible and would force the user to install a dummy image just to get passed this section of the script. Do you think this is okay or should we instead throw a message but let execution continue?

Reply:
my understanding, or at least my guess is that if a user's docker daemon is running but they don't have any installed/loaded images and they run `docker images`, they should get some feedback but it should not error out, and therefore should not trigger an exit code that will cause the `if` statement to run.

Reply:
Oh, that makes sense. Thanks!